internal/e2e/updatetest/deb_test.go

```diff
@@ -0,0 +1,122 @@
+package updatetest
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+var arch = flag.String("arch", "amd64", "target architecture")
```

Comment:
Isn't this just 
```suggestion
var arch = runtime.GOARCH
```

I would expect that you want to install/build the artifact with the same architecture of the current machine